pyproject.toml

```diff
@@ -37,3 +37,4 @@ markers = [
     "podman: marks tests as needing a podman engine to run"
 ]
 xfail_strict = true
+pythonpath = ["./"]
```

Comment:
This makes just `pytest` work from the project root, rather than needing `python3 -m pytest`